### PR TITLE
feat(rn): Update to new one-line command to install RN SDK

### PR DESCRIPTION
**Pull request by krystofwoldrich**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: src/platform-includes/getting-started-install/react-native.mdx [hunk 1]
```diff
@@ -4,37 +4,30 @@ If you wish to install Sentry's React Native SDK v4, follow the steps [here](/pl
...
-
-cd ios
-pod install
+yarn sentry-wizard -s -i reactNative
 ```
 
 <Note>
 
-For React Native 0.68 and older, you'll have to manually patch the Xcode project. Read more on the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
+For React Native 0.68 and older, you'll have to `confirm` to use wizard on unsupported React Native version and manually patch the Xcode project later. Read more on the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
```
Comment (shanamatthews) [inline src/platform-includes/getting-started-install/react-native.mdx:19]:
```suggestion
The wizard doesn't support React Native versions `0.68` and older. To use the wizard with older versions of React Native, you'll have to `confirm` in the wizard dialogue and manually patch the Xcode project later. Read more on the [Manual Configuration](/platforms/react-native/manual-setup/manual-setup/) page.
```

File: src/platform-includes/getting-started-install/react-native.mdx [hunk 2]
```diff
@@ -4,37 +4,30 @@ If you wish to install Sentry's React Native SDK v4, follow the steps [here](/pl
...
 
 </Note>
 
 [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project accordingly, though you can [set up manually](/platforms/react-native/manual-setup/manual-setup/) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
 The following tasks will be performed by the Sentry Wizard:
 
+- Install `@sentry/react-native` package.
```
Comment (shanamatthews) [inline src/platform-includes/getting-started-install/react-native.mdx:26]:
```suggestion
- Install the `@sentry/react-native` package.
```

**Review by shanamatthews — APPROVED** [review]
Small language tweaks again :D

**Comment by krystofwoldrich** [comment]
Thank you 🙏

**Pull request merged**